web_src/js/features/repo-issue-pr-status.js

```diff
@@ -0,0 +1,17 @@
+import $ from 'jquery';
+
+export function initRepoPullRequestCommitStatus() {
+  const $prStatus = $('.pr-status');
+
+  $('.hide-all-checks').on('click', async (e) => {
+    e.preventDefault();
+    const $this = $(e.currentTarget);
+
+    if ($prStatus.hasClass('hide')) {
+      $this.text($this.attr('data-hide-all'));
+    } else {
+      $this.text($this.attr('data-show-all'));
+    }
+    $prStatus.toggleClass('hide');
+  });
+}
```

Comment:
```suggestion
export function initRepoPullRequestCommitStatus() {
  const hideButton = document.querySelector('.hide-all-checks');
  if (!hideButton) return;

  hideButton.addEventListener('click', (e) => {
    e.preventDefault();
    const btn = e.currentTarget;

    if (btn.classList.contains('hide')) {
      btn.textContent = btn.getAttribute('data-hide-all');
      btn.classList.remove('hide');
    } else {
      btn.textContent = btn.getAttribute('data-show-all');
      btn.classList.add('hide');
    }
  });
}
```

Untested, please try. We should not be using jQuery any more for new features.

Reply:
Done in https://github.com/go-gitea/gitea/pull/26284/commits/9e4dfa54d983aa3d091c5be007a604064d10fb67